docs/user/README.md

```diff
@@ -202,14 +206,41 @@ Selects the line including the token containing letter 'a' with a blue hat.
 
 ##### `"file"`
 
-The word file can be used to expand the target to refer to the entire file.
+The word '`"file"` can be used to expand the target to refer to the entire file.
 
 - `"copy file"`
 - `"take file"`
 - `"take file blue air"`
 
 For example, `"take file [blue] air"` selects the file including the token containing letter 'a' with a blue hat.
 
+##### `"head & tail"`
+
+The words `"head"` and `"tail"` can be used to create a selection from the cursor's position, through the beginning or end of the line, respectively.
+
+- `"take head"`: select from the cursor the start of the line
+- `"take tail"`: select from the cursor to the end of the line
+- `"take head spun"`
+- `"take tail spun"`
+
+When you have a single cursor point and issue the first two commands, a selection will be made from that position to either the beginning or end of the file. If you have any code highlighted or address a mark(see last two examples) a selection joining the existing selection plus either the beginning or end of the line.
```

Comment:
Not sure these lines help, and I think are a bit confusing.  Prob better to just add a note to the "spun" lines above